Mammoth/TSE/CInstalledDevice.cpp

```diff
@@ -560,7 +579,7 @@ void CInstalledDevice::PaintDevicePos (const SDeviceDesc &Device, CG32bitImage &
 		}
 	}
 
-void CInstalledDevice::ReadFromStream (CSpaceObject &Source, SLoadCtx &Ctx)
+void CInstalledDevice::ReadFromStream (CSpaceObject &Source, SLoadCtx& Ctx)
```

Comment:
I know this is modern C++ syntax, but I'd like to keep it consistent with the original style.